### PR TITLE
fix: create secrets during upgrades (PROJQUAY-7001)

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

**Review by harishsurf — APPROVED** [review]
lgtm!

File: ansible-runner/context/app/project/roles/mirror_appliance/tasks/migrate-pre-1_3_11.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- name: Create Postgres Password Secret
+  containers.podman.podman_secret:
+    state: present
+    name: pgdb_pass
```
Comment (harishsurf) [inline ansible-runner/context/app/project/roles/mirror_appliance/tasks/migrate-pre-1_3_11.yaml:4]: nit: do we want to replace it with `{{ pgdb_password }}` defined in [secret-vars.yaml](https://github.com/quay/mirror-registry/blob/main/ansible-runner/context/app/project/roles/mirror_appliance/tasks/secret-vars.yaml#L5)

**Review by harishsurf — APPROVED** [review]

**Comment by BadgerOps** [comment]
Hey @harishsurf @dmage sorry for this regression - I'm just wrapping up some local testing for a similar PR that generates facts :

```
- name: Include vars of the config.yaml into the 'quay_config' variable (2.2).
  ansible.builtin.include_vars:
    file: "{{ quay_root }}/quay-config/config.yaml"
    name: quay_config
 ``` 
 
 Then creates the podman secrets with those - the reason I was thinking of going this route is to protect backwards compatibility. Of course, if the better option is to reset the passwords to what is autogenerated in `secret-vars.yaml` then I'm all for that, and I'll toss my branch :rofl:

**Comment by BadgerOps** [comment]
I just submitted https://github.com/quay/mirror-registry/pull/151 with another way - let me know if y'all would prefer to do it another way and I'll close mine!

**Comment by dmage** [comment]
Replaced by #151.

**Pull request closed without merging**